README.md

```diff
@@ -174,6 +174,125 @@ Which should print a transcription of the audio file. If you would like to strea
 swift run whisperkit-cli transcribe --model-path "Models/whisperkit-coreml/openai_whisper-large-v3" --stream
 ```
 
+### WhisperKit Local Server
+
+WhisperKit includes a local server that implements the OpenAI Audio API, allowing you to use existing OpenAI SDK clients or generate new ones. The server supports transcription and translation with streaming capabilities.
```

Comment:
Let's clarify "streaming capabilities" as output streaming only. Could be misinterpreted as full-duplex streaming like  the Pro version: https://www.argmaxinc.com/blog/argmax-local-server

Reply:
Actually, would be useful to add a callout block and link out to the Pro Local Server page for real-time transcription server